.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
```

Comment:
Looks like v4.5.0 is the latest version. Is there a reason we're on an older one?

Reply:
I tried a newer version and it failed, but I didn't try 4.5.0. Let's see.